$NetBSD: patch-syncserver-db-common_Cargo.toml,v 1.1 2024/06/13 12:11:59 wiz Exp $

Let pkgsrc provide the file instead of running git to fetch.

--- syncserver-db-common/Cargo.toml.orig	2024-05-31 23:17:56.000000000 +0000
+++ syncserver-db-common/Cargo.toml
@@ -11,7 +11,7 @@ futures.workspace = true
 http.workspace = true
 thiserror.workspace = true
 
-deadpool = { git = "https://github.com/mozilla-services/deadpool", tag = "deadpool-v0.7.0" }
+deadpool = { path = "../../deadpool" }
 diesel = { version = "1.4", features = ["mysql", "r2d2"] }
 diesel_migrations = { version = "1.4.0", features = ["mysql"] }
 syncserver-common = { path = "../syncserver-common" }
